src/main/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBFToSMC.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.digital.ho.hocs.casework.api.factory.strategies;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.CaseDataService;
+import uk.gov.digital.ho.hocs.casework.api.CorrespondentService;
+import uk.gov.digital.ho.hocs.casework.api.factory.CaseCopy;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+
+@Service
+@CaseCopy(fromCaseType = "BF", toCaseType = "SMC")
+public class CopyBFToSMC extends AbstractCaseCopyStrategy implements CaseCopyStrategy {
```

Comment:
Are you writing any tests for this strategy @JosephRogerson?

Reply:
I've added a test for this strategy